src/components/ui/email-input/email-input.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { UseFormReturn } from 'react-hook-form';
+import { InputValidationError } from '../input-validation-error';
+
+type Props = {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
any is not good😁

Reply:
for now typed it as useForm return
![image](https://github.com/user-attachments/assets/ef86b91a-8f76-495b-92b9-6af16da50ed1)